src/components/media-video.js

```diff
@@ -521,7 +521,9 @@ AFRAME.registerComponent("media-video", {
         // We want to treat audio almost exactly like video, so we mock a video texture with an image property.
         texture = new THREE.Texture();
         texture.image = videoEl;
-        isReady = () => true;
+        isReady = () => {
+          return videoEl.readyState > 0 ? true : false;
```

Comment:
nit: The ternary seems redundant.
```suggestion
          return videoEl.readyState > 0;
```